impl/src/fmt/mod.rs

```diff
@@ -671,14 +674,17 @@ trait FieldsExt {
     /// [`FmtAttribute`] as [`FmtArgument`]s or named [`Placeholder`]s.
     ///
     /// [`syn::Ident`]: struct@syn::Ident
-    fn fmt_args_idents(&self) -> impl Iterator<Item = syn::Ident> + '_;
+    // TODO: Return `impl Iterator<Item = syn::Ident> + '_` once MSRV is bumped up to 1.75 or
+    //       higher.
```

Comment:
I'm fine with bumping MSRV to 1.75 for this

Reply:
@JelteF I'm fine too, but better do it as a separate PR then.

Reply:
I'll do shortly after merging this one.